### PR TITLE
[NUI] Implement new window policy decision APIs of webview.

### PR DESCRIPTION
**Pull request by huayongxu**
This patch is to implement WebView APIs for new window policy
decision.

### API Changes ###

 - ACR:

### CODE REVIEW

**Comment by huayongxu** [comment]
This patch is related to the patch at csharp-binder side:
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/252959/

**Review by dongsug-song — COMMENTED** [review]
looks good to me, but other member needs review this also.

**Comment by dongsug-song** [comment]
@huayongxu 
could you resolve the error below?
**"This branch has conflicts that must be resolved"**

File: src/Tizen.NUI/src/internal/WebView/WebNewWindowPolicyDecisionMaker.cs [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+        }
+
+        /// <summary>
+        /// Decision type
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum DecisionType
+        {
+            /// <summary>
+            /// Use
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/internal/WebView/WebNewWindowPolicyDecisionMaker.cs:40]: What is `Use` in terms of the new window policy?

**Review by rabbitfor — APPROVED** [review]
Mr. Xu. Please add more description for the enum values in `WebNewWindowPolicyDecisionMaker` as I commented if possible. Otherwise the patch looks good to me. Thank you.

**Review by zg2nets — APPROVED** [review]
looks good to me

File: src/Tizen.NUI/src/internal/WebView/WebNewWindowPolicyDecisionMaker.cs (hunk 1 above)
Comment (huayongxu) [reply src/Tizen.NUI/src/internal/WebView/WebNewWindowPolicyDecisionMaker.cs]: Done.

**Comment by huayongxu** [comment]
> @huayongxu
> could you resolve the error below?
> **"This branch has conflicts that must be resolved"**

Done.

**Pull request merged**